src/components/AdvertForm/AdvertForm.js

```diff
@@ -45,13 +47,12 @@ class AdvertForm extends PureComponent {
                     <input type="number" value={this.state.price} onChange={this.handlePriceChange}/>
                 </label>
 
-                <button onClick={(event) => onAddAdvert(this.state)}>AddAdvert</button>
+                <button onClick={() => onAddAdvert(this.state)}>AddAdvert</button>
```

Comment:
https://reactjs.org/docs/handling-events.html read pls.
In this case, you create new function each time {() => onAddAdvert(this.state)} . And check all places.